src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -2228,7 +2260,32 @@ async def CommissionOnNetwork(self, nodeId: int, setupPinCode: int,
                     self.devCtrl, self.pairingDelegate, nodeId, setupPinCode, int(filterType), str(filter).encode("utf-8") if filter is not None else None, discoveryTimeoutMsec)
             )
 
-            return await asyncio.futures.wrap_future(ctx.future)
+            # If RCAC data is needed, await the result
+            if get_rcac:
+                try:
+                    sleep(60)
```

Comment:
why is this required?

Reply:
I was able to remove this sleep after moving the return await asyncio.futures.wrap_future(ctx.future) to above where the rcac_bytes variable gets set in this CommissionOnNetwork().